docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -170,6 +173,39 @@ afterAll(({ params }) => {
 });
 ----
 
+[discrete]
+[[monitor-use]]
+== Configure monitors
+
+// Does it make sense to include this here or should it be included in
+// Run synthetic tests > Use a test suite > Send to {stack}
+// since we don't know what a push command is yet?
+
+[ Configure monitor-related fields using the Monitor DSL. ]
+
+[source,js]
+----
+import { journey, step, monitor, expect } from '@elastic/synthetics';
+
+journey('My Example Journey', ({ page, params }) => {
+  /**
+   *  Only relevant for the push command to create monitors in Kibana
+   */
+  monitor.use({
+    id: 'example-monitor',
+    schedule: 10,
+    throttling: {
+      download: 10,
+      upload: 5,
+      latency: 100,
+    },
+  });
+  step('launch application', async () => {
+    await page.goto('https://example.com');
+  });
+});
+----
+
 [discrete]
 [[synthetics-sample-test]]
 == Sample synthetic test
```

Comment:
Depending on the changes in https://github.com/elastic/observability-docs/pull/1925/files#r892104623 it might be worth expanding this example to include the new `monitor.use...` config instead